File: 프로그래머스/unrated/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.cs
using System;

public class Solution {
    public int[] solution(int[] arr, int[,] queries) 
    {
        for(int y = 0; y < queries.GetLength(0); ++y)
        {
            int s = queries[y, 0];
            int e = queries[y, 1];
            int k = queries[y, 2];
            
            for(int i = s; i <= e; ++i)
            {
                if(i % k == 0)
                    ++arr[i];
            }
        }
        return arr;
    }
}

File: 프로그래머스/0/181922. 수열과 구간 쿼리 4/수열과 구간 쿼리 4.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr, vector<vector<int>> queries) {
    
    for (int i = 0; i < queries.size(); ++i)
    {
        int s = queries[i][0];
        int e = queries[i][1];
        int k = queries[i][2];
        
        for (int i = s; i <= e; ++i)
        {
            if (i % k == 0)
                ++arr[i];
        }
    }
    
    return arr;
}